bundles/org.openhab.binding.gree/src/main/resources/ESH-INF/i18n/gree.properties

```diff
@@ -1,5 +1,6 @@
 # GREE Binding
 binding.gree.name = GREE Binding
+binding.gree.label = GREE Air Conditioner
```

Comment:
The binding is not limited to Air Conditioner, this is a thing type
where is the label required?